spec/requests/swagger_spec.rb

```diff
@@ -806,39 +806,82 @@
         expect(subject).to validate(:post, '/v0/hca_attachments', 400, '')
       end
 
-      it 'supports submitting a health care application', run_at: '2017-01-31' do
-        VCR.use_cassette('hca/submit_anon', match_requests_on: [:body]) do
+      context "when the 'va1010_forms_enrollment_system_service_enabled' flipper is enabled" do
+        it 'supports submitting a health care application', run_at: '2017-01-31' do
+          VCR.use_cassette('hca/submit_anon', match_requests_on: [:body]) do
+            expect(subject).to validate(
+              :post,
+              '/v0/health_care_applications',
+              200,
+              '_data' => {
+                'form' => test_veteran
+              }
+            )
+          end
+
           expect(subject).to validate(
             :post,
             '/v0/health_care_applications',
-            200,
+            422,
+            '_data' => {
+              'form' => {}.to_json
+            }
+          )
+
+          allow_any_instance_of(HCA::Service).to receive(:submit_form) do
+            raise Common::Client::Errors::HTTPError, 'error message'
+          end
```

Comment:
This is different than what we had before adding the new enrollment service. The code on line `872` did not work. I'm wondering why we didn't allow it to receive `submit_form` in the original implementation instead of `:post` 🤔 This seems much more straight forward 🤷‍♂️ Please let me know if I'm missing something.